pkgs/by-name/id/ideviceinstaller/package.nix

```diff
@@ -1,42 +1,47 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, autoreconfHook
-, pkg-config
-, usbmuxd
-, libimobiledevice
-, libzip
-}:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  unstableGitUpdater,
+
+  autoreconfHook,
+  pkg-config,
 
-stdenv.mkDerivation rec {
+  libimobiledevice,
+  libzip,
+  usbmuxd,
+}:
+stdenv.mkDerivation (finalAttrs: {
   pname = "ideviceinstaller";
-  version = "1.1.1+date=2023-04-30";
+  version = "1.1.1-unstable-2024-05-18";
 
   src = fetchFromGitHub {
     owner = "libimobiledevice";
-    repo = pname;
-    rev = "71ec5eaa30d2780c2614b6b227a2229ea3aeb1e9";
-    hash = "sha256-YsQwAlt71vouYJzXl0P7b3fG/MfcwI947GtvN4g3/gM=";
+    repo = "ideviceinstaller";
+    rev = "1431d42b568ee78161a41ed02df0de60dc1439d6";
+    hash = "sha256-aXnh2ydukKILPhLv4eSu73IUEZhpin8abaw9e4UCTRk=";
   };
 
+  passthru.updateScript = unstableGitUpdater { };
+
   nativeBuildInputs = [
     autoreconfHook
     pkg-config
   ];
 
   buildInputs = [
-    usbmuxd
     libimobiledevice
     libzip
+    usbmuxd
   ];
 
   # the package uses zip_get_num_entries, which is deprecated
-  env.NIX_CFLAGS_COMPILE = toString [
+  NIX_CFLAGS_COMPILE = toString [
```

Comment:
```suggestion
  env.NIX_CFLAGS_COMPILE = toString [
```

Reply:
Whoops, seems I've made a change that isn't the best. Could I ask why this is preferred? Afaict both will propagate as an env var for the derivation. Is this preferred due to its more explicit use of "env" to imply "as an env var"?